app/modules/phonebook/types_phonebook.py

```diff
@@ -1,15 +1,15 @@
 from enum import Enum
 
 
-class RoleTags(Enum):
+class RoleTags(str, Enum):
```

Comment:
We are not using this Enum as a string so I guess we don't need to inherit from str.